eng/pipelines/coreclr/templates/crossgen-comparison-job.yml

```diff
@@ -10,6 +10,13 @@ parameters:
   variables: {}
   pool: ''
 
+  # When set to a non-empty value (Debug / Release), it determines libraries
+  # build configuration to use for the tests. Setting this property implies
+  # a dependency of this job on the appropriate libraries build and is used
+  # to construct the name of the Azure artifact representing libraries build
+  # to use for building the tests.
+  liveLibrariesBuildConfig: ''
```

Comment:
I see you've also made the live-live crossgen comparison change. For now I assume that you're going to merge it in together with your other changes unless you tell me otherwise.